repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -128,6 +127,38 @@ public List<AtlasTask> getInProgressTasks() {
         return ret;
     }
 
+    public List<AtlasTask> getInProgressTasksES() {
+        List<AtlasTask> ret = new ArrayList<>();
```

Comment:
Please add metric logging here to track getInProgressTasksES

Reply:
Does it make sense to paginate here rather than having one giant ArrayList? Also should we do some sort of priority here?